File: CLASE II - TDA/clase_2.2_ayed/clase_1_ayed/c/main.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

#include "persona.h"
#include "carton.h"


///Persona anidada con carton, CON TDA

int main()
{
    srand(time(0));
    printf("------ Clase 1 - AYED - REPASO -------\n");


    PersonaPtr jugador = crearPersonaYCarton(123, "Nico", "Av. Peron 1130" );
    mostrarPersona(jugador);

    //Quiero cambir algo del carton
    setVecPos(getCartonPtr(jugador),2, 99);
    mostrarPersona(jugador);

    int vector[8] = {2,63,96,6,3,2,12,1};

    setVec(getCartonPtr(jugador),vector);
    mostrarPersona(jugador);


    return 0;
}

File: CLASE II - TDA/clase_2.2_ayed/clase_1_ayed/golang/main.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

func main() {
	rand.Seed(time.Now().UnixNano())
	fmt.Printf("------ Clase 1 - AYED - REPASO -------\n")

	jugador := crearPersonaYCarton(123, "Nico", "Av. Peron 1130")
	mostrarPersona(jugador)

	// Quiero cambiar algo del carton
	jugador.GetCartonPtr().SetVecPos(2, 99)
	mostrarPersona(jugador)

	vector := [8]int{2, 63, 96, 6, 3, 2, 12, 1}
	jugador.GetCartonPtr().SetVec(vector)
	mostrarPersona(jugador)
}

